requirements.txt

```diff
@@ -0,0 +1 @@
+django-appconf
```

Comment:
This is missing from `requirements-dev.txt`. `Django` is also a dependency, but not specified in either requirements file. Not sure if oversight or on purpose?

Reply:
requirements-dev contains `-e .` which does install all requirements form the package.
Django is always omitted, I don't know why people do that really. But I go with it.